server/routes/reviewsHelpers.js

```diff
@@ -122,18 +125,74 @@ const addHelpfulVote = (reviewId) => {
       return data.status
     })
     .catch((err) => {
-      console.error('ERROR in addHelpfulVote: ', err);
+      console.error('ERROR in addHelpfulVote: ', err)
     });
 }
 
+const formatPostData = (postData) => {
+  console.log('TWO')
+  return new Promise ((resolve, reject) => {
+    let formattedPostData = {};
+
+    formattedPostData['product_id'] = Number(postData.product_id);
+    formattedPostData['rating'] = Number(postData.rating);
+    formattedPostData['summary'] = postData.summary;
+    formattedPostData['body'] = postData.body;
+
+    let rec;
+    if (postData['recommend'] === 'true') {
+      rec = true;
+    } else if (postData['recommend'] === 'false') {
+      rec = false;
+    }
+
+    formattedPostData['recommend'] = rec;
+    formattedPostData['name'] = postData.name;
+    formattedPostData['email'] = postData.email;
+
+    let formattedChars = {};
+    let postChars = postData.characteristics;
+
+    for (key in postChars) {
+      if (postChars[key].id !== '0') {
+        formattedChars[postChars[key].id] = Number(postChars[key].rating)
+      }
+    }
+
+    formattedPostData['characteristics'] = formattedChars;
+    resolve(formattedPostData);
```

Comment:
Good job resolving on the value you want. You'll need a reject in here as well in a catch block. If this function errors out for any reason there is nothing to handle that at the moment.